docs/examples/gluster-s3-storage-template/README.md

```diff
@@ -2,111 +2,136 @@
 
 ## Prerequisites
 
-  OpenShift setup is up with master and nodes ready.
+* OpenShift setup is up with master and nodes ready.
 
-  cns-deploy tool is ran and heketi service is ready
+* cns-deploy tool has been run and heketi service is ready.
 
+## Deployment
 
+### 1. Provide the backend store
 
+The gluster-s3 service requires there be at least two GlusterFS volumes
+available for its use, one to store the object data and another for the
+object meta-data. In this example, we will create a new StorageClass to
+dynamically provision these two volumes on our pre-existing GlusterFS cluster.
 
-## First create a storageclass
+#### Create a StorageClass
 
-Create a GlusterFS StorageClass as below, replacing the `rest` parameters with your configuration:
+In our example, we have set up heketi to require a secret key for the admin user. A StorageClass created to use such a heketi instance needs a Secret that contains the admin key. This Secret is not needed if heketi is not configured to use an admin key.
+
+ Replace `NAMESPACE` and `ADMIN_KEY` parameters with your configuration.
+Here, `NAMESPACE` is the project and `ADMIN_KEY` is used for authorization to access Heketi service.
```

Comment:
Remove the newline on the previous line.

Reply:
Done